driver/driver.go

```diff
@@ -36,9 +37,9 @@ type Driver struct {
 	l1HeadSub  event.Subscription
 	syncNotify chan struct{}
 
-	backOffRetryInterval time.Duration
-	ctx                  context.Context
-	wg                   sync.WaitGroup
+	RetryInterval time.Duration
```

Comment:
This value is not initialized in `InitFromConfig`?

Reply:
Oh, `RetryInterval` should be deleted just use the same name in `*Config` is enough.